### ISSUE TITLE
Importing ABC directly from collections was removed in Python 3.9

### ISSUE THREAD

**Issue by tirkarthi** [tensorflow/tensorboard#3202]
# Issue description

Importing ABC directly from collections was removed in Python 3.9 use collections.abc instead. Below are instances

```
python
Python 3.8.0 (default, Nov  6 2019, 21:49:08) 
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import collections
>>> collections.Callable
<stdin>:1: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.9 it will stop working
<class 'collections.abc.Callable'>
```

```
rg "collections.*(Awaitable|Coroutine|AsyncIterable|AsyncIterator|AsyncGenerator|Hashable|Iterable|Iterator|Generator|Reversible|Sized|Container|Callable|Collection|MutableSet|Mapping|MutableMapping|MappingView|KeysView|ItemsView|ValuesView|Sequence|MutableSequence|ByteString)\b" 
tensorboard/summary/summary_test.py
43:                getattr(self.module, plugin), collections.Callable
49:                getattr(self.module, "%s_pb" % plugin), collections.Callable

tensorboard/plugins/scalar/scalars_plugin_test.py
188:        self.assertIsInstance(routes["/scalars"], collections.Callable)
189:        self.assertIsInstance(routes["/tags"], collections.Callable)

tensorboard/plugins/histogram/histograms_plugin_test.py
159:        self.assertIsInstance(routes["/histograms"], collections.Callable)
160:        self.assertIsInstance(routes["/tags"], collections.Callable)

tensorboard/plugins/image/images_plugin_test.py
170:        self.assertIsInstance(self.routes["/images"], collections.Callable)
172:            self.routes["/individualImage"], collections.Callable
174:        self.assertIsInstance(self.routes["/tags"], collections.Callable)

tensorboard/plugins/mesh/mesh_plugin_test.py
173:        self.assertIsInstance(self.routes["/tags"], collections.Callable)
174:        self.assertIsInstance(self.routes["/meshes"], collections.Callable)
175:        self.assertIsInstance(self.routes["/data"], collections.Callable)

tensorboard/plugins/debugger/events_writer_manager_test.py
40:        self.assertIsInstance(apps["/health_pills"], collections.Callable)

tensorboard/plugins/pr_curve/pr_curves_plugin_test.py
124:        self.assertIsInstance(routes["/tags"], collections.Callable)
125:        self.assertIsInstance(routes["/pr_curves"], collections.Callable)
127:            routes["/available_time_entries"], collections.Callable

tensorboard/plugins/audio/audio_plugin_test.py
146:        self.assertIsInstance(routes["/audio"], collections.Callable)
147:        self.assertIsInstance(routes["/individualAudio"], collections.Callable)
148:        self.assertIsInstance(routes["/tags"], collections.Callable)

tensorboard/plugins/distribution/distributions_plugin_test.py
118:        self.assertIsInstance(routes["/distributions"], collections.Callable)
119:        self.assertIsInstance(routes["/tags"], collections.Callable)

tensorboard/plugins/graph/graphs_plugin_test.py
138:        self.assertIsInstance(routes["/graph"], collections.Callable)
139:        self.assertIsInstance(routes["/run_metadata"], collections.Callable)
140:        self.assertIsInstance(routes["/info"], collections.Callable)

tensorboard/plugins/debugger/debugger_plugin_test.py
40:        self.assertIsInstance(apps["/health_pills"], collections.Callable)

tensorboard/plugins/core/core_plugin_test.py
163:        self.assertIsInstance(routes["/data/logdir"], collections.Callable)
164:        self.assertIsInstance(routes["/data/runs"], collections.Callable)

tensorboard/plugins/text/text_plugin_test.py
55:        self.assertIsInstance(routes["/tags"], collections.Callable)
56:        self.assertIsInstance(routes["/text"], collections.Callable)

```

**Comment by wchargin** [comment]
Hi @tirkathi! Thanks for pointing this out. Fixed in #3204.

**Comment by tirkarthi** [comment]
Thanks @wchargin :)
